src/useReactions.tsx

```diff
@@ -92,19 +99,38 @@ export const ReactionsProvider = ({
     clientState?.state === "valid" && clientState.supportedFeatures.reactions;
   const room = rtcSession.room;
   const myUserId = room.client.getUserId();
+  const myDeviceId = room.client.getDeviceId();
```

Comment:
Potentially this hook could be split up, and listening for incoming reactions could be moved somewhere else (maybe even MatrixRTCSession?).

Reply:
I've split the sending and receiving logic pieces into a provider and a producer-style hook, using the call view model for state.